src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -342,6 +342,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 });
             }
 
+            if(env.IsDevelopment() || !String.IsNullOrEmpty(Environment.GetEnvironmentVariable("enableSwagger")))
```

Comment:
It depends on how you end up declaring the App Service parameter, but if it has a default value of false then it won't be a check for NullOrEmpty. Also you can use the type `string` here rather than `String`. That should be a code warning.